src/QuadForm/Quad/Spaces.jl

```diff
@@ -816,6 +824,9 @@ end
 # p is the prime idela
 function _can_locally_embed(n::Int, da, ha::Int, m::Int, db, hb::Int, p)
   de = m - n
+  if de < 0
+    return false
```

Comment:
Did you have an example for this?

Reply:
The logic should be clear if dim(M)=m > n=dim(N) then M cannot embedd in N

Reply:
Yes, but it is not covered by the tests.